src/components/Authorship.vue

```diff
@@ -0,0 +1,98 @@
+<template>
+  <div id="author-container">
+    <p>
+      <span id="primary-author-statment">
+        The development was lead by 
+        <span
+          v-for="(author, index) in primaryAuthors" 
+          :key="`${author.initials}-attribution`"
+          :id="`initial-${author.initials}`"
+          :class="'author first'"
+        >
+          <a v-bind:href="author.profile_link" target="_blank" v-text="author.fullName"></a>
+          <span v-if="index != Object.keys(primaryAuthors).length - 1 && Object.keys(primaryAuthors).length > 2">, </span>
+          <span v-if="index == Object.keys(primaryAuthors).length - 2"> and </span>
+        </span>.
+      </span>
+      <span id="additional-author-statement" v-if="showAdditionalAuthors">
+        <span
+          v-for="(author, index) in additionalAuthors" 
+          :key="`${author.initials}-attribution`"
+          :id="`author-${author.initials}`"
+          :class="'author'"
+        >
+          <a v-bind:href="author.profile_link" target="_blank" v-text="author.fullName"></a>
+          <span v-if="index != Object.keys(additionalAuthors).length - 1 && Object.keys(additionalAuthors).length > 2">, </span>
+          <span v-if="index == Object.keys(additionalAuthors).length - 2"> and </span>
+        </span>
+        <span>
+        also contributed to the site.
+        </span>
+      </span>
+      <span id="contribution-statements" v-if="showContributionStatements">
+        <span id="primary-author-contribution">
+          <span
+            v-for="author in primaryAuthors" 
+            :key="`${author.initials}-contribution`"
+            :id="`author-${author.initials}`"
+            :class="'author'"
+          >
+            <span v-text="author.firstName"> </span> <span v-text="author.contribution"></span>. 
+          </span>
+        </span>
+        <span id="additional-author-contribution"  v-if="showAditionalContributionStatement">
+          <span
+            v-for="author in additionalAuthors" 
+            :key="`${author.initials}-contribution`"
+            :id="`author-${author.initials}`"
+            :class="'author'"
+          >
+            <span v-text="author.firstName"> </span> <span v-text="author.contribution"></span>. 
+          </span>
+        </span>
+      </span>
+    </p>
+  </div>
+</template>
+
+<script>
+import { isMobile } from 'mobile-device-detect';
+import authors from "@/assets/text/authors";
+export default {
+  name: "authorship",
+    components: {
+    },
+    props: {
+    },
```

Comment:
could use props to import the title from the parent component

Reply:
K just added - ended up just pulling the `VUE_APP_TITLE` environment variable that's set in `.env`